.github/actions/ghcr-exists/action.yml

```
@@ -0,0 +1,53 @@
+name: Check if a tag exists on GHCR for a given image
+
+inputs:
+  image_name:
+    description: 'The name of the image (for example "aica-technology/image-name")'
+    required: true
+    type: string
+  image_tag:
+    description: 'The tag of the image (for example "devel")'
+    required: true
+    type: string
+  token:
+    description: "The GitHub token passed from the caller workflow"
+    required: true
+
+outputs:
+  exists:
+    description: "Does it exists?"
```

the nittiest of picks
```suggestion
    description: "Does it exist?"
```